.changeset/beige-nails-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hashicorp/react-call-to-action': minor
+---
+
+Add support for brand theme. Fix issue with custom content rendering.
```

Comment:
Example use case for brand theme (via [www-next#2303](https://github.com/hashicorp/hashicorp-www-next/pull/2303)): [/partners/become-a-partner](https://hashicorp-www-next-git-zscss-modules-hashicorp.vercel.app/partners/become-a-partner), at bottom of page.